frontend/test/pytest/test_template.py

```diff
@@ -63,7 +94,53 @@ def basis_embedding(f: jax.core.ShapedArray([3], int)):
     params = jax.numpy.array([1, 1, 1])
     interpreted_fn = qml.QNode(basis_embedding, device)
     jitted_fn = qjit(interpreted_fn)
-    assert np.allclose(interpreted_fn(params), jitted_fn(params))
+
+    interpreted_result = interpreted_fn(params)
+    jitted_result = jitted_fn(params)
+
+    assert np.allclose(interpreted_result, jitted_result)
+
+
+@pytest.mark.xfail(reason="Displacement operator not supported on lightning.")
```

Comment:
I think quantum optics is no longer in pennylane's scope anymore? @josh146 @dime10

Reply:
Anything that inherits from CVOperation can be ignored --- the only device these are supported on is `default.gaussian` and it is likely not even tested anymore, and definitely not with JIT compilation.